Unity/Assets/MyScripts/MyAppStart.cs

```diff
@@ -29,67 +36,84 @@ void Start ()
 
 		Debug.Log ("unity-script: unity version" + IronSource.unityVersion ());
 
-		// Add Banner Events
-		IronSourceBannerEvents.onAdLoadedEvent += BannerOnAdLoadedEvent;
-		IronSourceBannerEvents.onAdLoadFailedEvent += BannerOnAdLoadFailedEvent;
-		IronSourceBannerEvents.onAdClickedEvent += BannerOnAdClickedEvent;
-		IronSourceBannerEvents.onAdScreenPresentedEvent += BannerOnAdScreenPresentedEvent;
-		IronSourceBannerEvents.onAdScreenDismissedEvent += BannerOnAdScreenDismissedEvent;
-		IronSourceBannerEvents.onAdLeftApplicationEvent += BannerOnAdLeftApplicationEvent;
-
 		// SDK init
 		Debug.Log ("unity-script: IronSource.Agent.init");
```

Comment:
this log should be removed or updated as we are no longer init via IronSource.Agent.

Reply:
edited